File: logical-operator/2-1.py
input_id = input("please enter your ID.\n")
input_pwd = input("please enter your password.\n")
real_id = "egoing"
real_pwd = "11"
if real_id == input_id:
    if real_pwd ==input_pwd:
        print("Hello!")
    else:
        print("Wrong password.")
else:
    print("Wrong id.")

File: logical-operator/2-1.rb
puts("please enter your id.")
input_id = gets.chomp()
puts("please enter your password.")
input_pwd = gets.chomp()
real_id = "egoing"
real_pwd = "ab"
if real_id == input_id and real_pwd == input_pwd
    puts("Hello")
else
    puts("Failed to log in")
end